Euroleague-Graduate/Euroleague-Graduate/TurkishAirLinesHelper.swift

```diff
@@ -15,49 +17,171 @@ class TurkishAirLinesHelper {
         case results
     }
     
+    //In viewModel has array of games according to target round
+    var table: Dictionary<String, [Array<GameData>]> = [:]
+    
+    private var games: Dictionary<Int, GameData> = [:] {
+        didSet{
+            //Notify the view Model
+        }
+    }
+    
+    //In viewModel has target round
+    private var round: String = ""
+    
     let urls: Dictionary<GameDataStatus, String> = [
         .schedule : "http://www.euroleague.net/euroleague/api/schedules?seasoncode=E2016",
         .results : "http://www.euroleague.net/euroleague/api/results?seasoncode=E2016"
     ]
     
-    func getGamesTable() {
+    //Should have no parameters and return table of dataBase
+    func getGamesTable(round: String) {
+        games.removeAll()
```

Comment:
There should be a data service layer to handle the parsing

ApiClient -> GamesApiClient -> GamesDataService